js/src/pages/settings/enhanced-conversions/useEnableEnhancedConversions.js

```diff
@@ -0,0 +1,20 @@
+/**
+ * External dependencies
+ */
+import { useSelect } from '@wordpress/data';
+
+/**
+ * Internal dependencies
+ */
+import { STORE_KEY } from '~/data';
+
+/**
+ * Custom hook to manage the state of enhanced conversions.
+ * @return {boolean} An array containing the current state and a function to toggle it.
+ */
+export const useEnableEnhancedConversions = () => {
```

Comment:
@ankitrox Nitpick, but let's rewrite the selector to follow the same format as other selectors:
```js
const selectorName = 'getEnableEnhancedConversions';

export const useEnableEnhancedConversions = () => {
	return useSelect( ( select ) => {
		const selector = select( STORE_KEY );
		
		return selector[ selectorName ]();
	}, [] );
};
```